rspace/src/main/scala/coop/rchain/rspace/history/HistoryStore.scala

```diff
@@ -35,7 +37,24 @@ object HistoryStoreInstances {
     override def get(key: Blake2b256Hash): F[Trie] =
       for {
         maybeBytes <- store.get(key)
```

Comment:
I know this is aesthetic but I have an allergy to these `maybe` names. 
Can we use `bytesOpt`?
Actually we should have everywhere `unsafeGet` extensions, so we don't have tons of such options across code that expresses useful logic.